pac4j-http/src/main/java/org/pac4j/http/credentials/CookieCredentials.java

```diff
@@ -0,0 +1,59 @@
+/*
+ *    Copyright 2012 - 2015 pac4j organization
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package org.pac4j.http.credentials;
+
+import org.pac4j.core.util.CommonHelper;
+
+/**
+ * @author Misagh Moayyed
+ * @since 1.8.1
+ */
+public class CookieCredentials extends HttpCredentials {
```

Comment:
I would have expected to use the `TokenCredentials` set with the cookie value (search based on the name)